screens/HomeScreen.js

```
@@ -234,6 +256,21 @@ const styles = StyleSheet.create({
 		opacity: 0
+	},
+	nativeCastButtonWrapper: {
+		position: 'absolute',
+		right: 16,
+		zIndex: 10
+	},
+	castButton: {
+		width: 24,
+		height: 24,
+		tintColor: '#fff'
+	},
```

ESLint

[error] Color literal: { tintColor: '#fff' }

(react-native/no-color-literals)